### DISCUSSION TITLE
experiencing synchronization error on grapheneOS

### DISCUSSION THREAD

**Discussion by b3K1ndRw1nd** [bitfireAT/icsx5#56] [General]
I see the following error on my Google Pixel 4 which is running grapheneOS

How can this be fixed?
![Screenshot_20220515-212939](https://user-images.githubusercontent.com/84428257/170943026-e03dedae-f745-4505-914d-71ffa0ac5762.png)

**Comment by rfc2822** [comment]
Hi,

Probably the same as https://github.com/bitfireAT/icsx5/discussions/24
